packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -122,8 +123,8 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
     }
     setCachedEditedVideoId(editedVideoInfo.id)
 
-    reset(tabData)
-  }, [tabData, tabDataLoading, reset, editedVideoInfo.id, cachedEditedVideoId, setValue])
+    reset({ ...tabData, mintNft: tabData.mintNft || !!mintNft })
```

Comment:
Not sure about this, `mintNft` will be set as a form default value. We have the cancel button and if you click it, all edits should be reset. However, `mintNft` will not have the proper value set. You could leave the `reset` as it is and add `setValue` after it to override. You probably will need to put `setValue` inside a `setTimeout`, otherwise it doesn't work